lib/blue_apron/spree_client.rb

```diff
@@ -107,8 +107,8 @@ def get_shipment(id, options = {})
       get "/api/shipments/#{id}", options
     end
 
-    def get_current_order_for(user_id)
-      get "/api/orders/current_for/#{user_id}"
+    def get_current_order_for(user_id, options = {})
+      get "/api/orders/current_for/#{user_id}", params: options
```

Comment:
All of the other methods above only pass in options as `get "/api/orders/current_for/#{user_id}", options` is there a reason not to do the same here?

Reply:
As far as I can tell via testing, that doesn't seem to work. It looks like that passes the options in as the body - which is weird on a get, and doesn't seem to come through in tests. I'm digging in a bit more, but...

Reply:
Is it possible that the higher up call is passing in `params: { foo: :bar }`? Or just the other methods never get called with options?

Reply:
Confirming with a quick test - when I called a SpreeClient instance `get_order` method, the "options" hash is ignored. Here's some results:

s.get_order('temp', { cart_type: 'blah' }) -> `params` in marketplace controller: none
s.get_order('temp', { cart_type: 'foo', params: { cart_type: 'bar' } }) -> params in controller: cart_type: 'bar'

So I think there's two options here:
1) Remain consistent with the other APIs in this gem, and require the client to pass `{ params: { cart_type: blah } }`
2) What I've done here. :-)

I can buy arguments either way - @ramimassoud , thoughts? I might lean more toward consistency after thinking this through, though.

Reply:
I always try to lean toward consistency since so often a lot of usage is "let me copy and paste this from over there that does something similar". I don't think it's just me that does that 😂 

If I had to guess, I'd say that having the client pass in a hash with `params` lets it also pass in some other options like maybe headers?

Reply:
Sounds good to me - updated!